WordPress/Classes/ViewRelated/Blog/DomainCredit/DomainCreditRedemptionSuccessViewController.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+@objc protocol DomainCreditRedemptionSuccessViewControllerDelegate {
```

Comment:
I'm wondering why the protocol is `@objc`. Since `continueButtonPressed` doesn't appear to be optional, would this be more Swifty? 
`protocol DomainCreditRedemptionSuccessViewControllerDelegate: class`

Reply:
Yes, looks more Swify without `@objc`. just updated!